webapp/src/store/Store.ts

```diff
@@ -18,4 +70,13 @@ export class Store {
     }
     return projectStore;
   }
+
+  public async loadFromDisk(): Promise<void> {
+    try {
+      const json = await fs.readFile(FILE_PATH);
+      this.initialState = JSON.parse(json.toString());
```

Comment:
`JSON.parse` could potentially execute for a long time if the input is very large and we never want to block Node's thread for long. Let's create a separate issue for exploring that and potentially finding an async replacement.